From 0e9e0b2a30e179612afeca4e44cb9224f864396f Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Tue, 2 Aug 2016 16:44:17 +0800
Subject: [PATCH 0395/1345] dts: a3700: fork the Z1 fdt files

commit  b6ed0a94a59d9a96d61fa4ea114b5f0ff7b75e63 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- The current kernel supports only Armada3700 Z1 chip/board. This
  patch is the preparation for Armada3700 A0 chip/board support.
- The dts/dtsi files for Z1 chip/board are duplicated from the
  current fdt files and renamed with the suffix of "rz" (shorten
  for "revision Z1"). The 'module' name in fdt has been updated
  accordingly.
- The existing fdt files are retained for A0 chip/board as the
  placeholders.
- Thhe dtb building commands for Armada 3720 Z1 boards should be:
  - make marvell/armada-3720-rz-db.dtb
  - make marvell/armada-3720-rz-db-sfp.dtb
  - make marvell/armada-3720-rz-db-sgmii1.dtb

Change-Id: Icb9aadc6cfb799972740519e08f36cb1c6389b18
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/31655
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Victor Gu <xigu@marvell.com>
Tested-by: Victor Gu <xigu@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/Makefile               |    3 +
 arch/arm64/boot/dts/marvell/armada-371x-rz.dtsi    |   53 +++++
 .../boot/dts/marvell/armada-3720-rz-db-sfp.dts     |   73 ++++++
 .../boot/dts/marvell/armada-3720-rz-db-sgmii1.dts  |   65 ++++++
 arch/arm64/boot/dts/marvell/armada-3720-rz-db.dts  |  243 ++++++++++++++++++++
 arch/arm64/boot/dts/marvell/armada-372x-rz.dtsi    |   63 +++++
 6 files changed, 500 insertions(+)
 create mode 100644 arch/arm64/boot/dts/marvell/armada-371x-rz.dtsi
 create mode 100644 arch/arm64/boot/dts/marvell/armada-3720-rz-db-sfp.dts
 create mode 100644 arch/arm64/boot/dts/marvell/armada-3720-rz-db-sgmii1.dts
 create mode 100644 arch/arm64/boot/dts/marvell/armada-3720-rz-db.dts
 create mode 100644 arch/arm64/boot/dts/marvell/armada-372x-rz.dtsi

diff --git a/arch/arm64/boot/dts/marvell/Makefile b/arch/arm64/boot/dts/marvell/Makefile
index 59f140e..36156ad 100644
--- a/arch/arm64/boot/dts/marvell/Makefile
+++ b/arch/arm64/boot/dts/marvell/Makefile
@@ -3,6 +3,9 @@ dtb-$(CONFIG_ARCH_BERLIN) += berlin4ct-dmp.dtb
 dtb-$(CONFIG_ARCH_BERLIN) += berlin4ct-stb.dtb
 
 # Mvebu SoC Family
+dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-rz-db.dtb
+dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-rz-db-sgmii1.dtb
+dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-rz-db-sfp.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-db.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-db-sgmii1.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-db-sfp.dtb
diff --git a/arch/arm64/boot/dts/marvell/armada-371x-rz.dtsi b/arch/arm64/boot/dts/marvell/armada-371x-rz.dtsi
new file mode 100644
index 0000000..bf2b9ea
--- /dev/null
+++ b/arch/arm64/boot/dts/marvell/armada-371x-rz.dtsi
@@ -0,0 +1,53 @@
+/*
+ * Device Tree Include file for Marvell Armada 371x family of RZ SoCs
+ * (also named 88F3710-RZ)
+ *
+ * Copyright (C) 2016 Marvell
+ *
+ * Gregory CLEMENT <gregory.clement@free-electrons.com>
+ *
+ * This file is dual-licensed: you can use it either under the terms
+ * of the GPL or the X11 license, at your option. Note that this dual
+ * licensing only applies to this file, and not this project as a
+ * whole.
+ *
+ *  a) This file is free software; you can redistribute it and/or
+ *     modify it under the terms of the GNU General Public License as
+ *     published by the Free Software Foundation; either version 2 of the
+ *     License, or (at your option) any later version.
+ *
+ *     This file is distributed in the hope that it will be useful
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ * Or, alternatively
+ *
+ *  b) Permission is hereby granted, free of charge, to any person
+ *     obtaining a copy of this software and associated documentation
+ *     files (the "Software"), to deal in the Software without
+ *     restriction, including without limitation the rights to use
+ *     copy, modify, merge, publish, distribute, sublicense, and/or
+ *     sell copies of the Software, and to permit persons to whom the
+ *     Software is furnished to do so, subject to the following
+ *     conditions:
+ *
+ *     The above copyright notice and this permission notice shall be
+ *     included in all copies or substantial portions of the Software.
+ *
+ *     THE SOFTWARE IS PROVIDED , WITHOUT WARRANTY OF ANY KIND
+ *     EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ *     OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ *     NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ *     HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY
+ *     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ *     OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include "armada-37xx.dtsi"
+
+/ {
+	model = "Marvell Armada 3710 RZ SoC";
+	compatible = "marvell,armada3710", "marvell,armada3700";
+};
diff --git a/arch/arm64/boot/dts/marvell/armada-3720-rz-db-sfp.dts b/arch/arm64/boot/dts/marvell/armada-3720-rz-db-sfp.dts
new file mode 100644
index 0000000..e46fe00
--- /dev/null
+++ b/arch/arm64/boot/dts/marvell/armada-3720-rz-db-sfp.dts
@@ -0,0 +1,73 @@
+/*
+ * Device Tree file for Marvell Armada 3720 RZ development board
+ * (DB-88F3720-DDR3-RZ)
+ * Copyright (C) 2016 Marvell
+ *
+ * Ken Ma <make@marvell.com>
+ *
+ * This file is dual-licensed: you can use it either under the terms
+ * of the GPL or the X11 license, at your option. Note that this dual
+ * licensing only applies to this file, and not this project as a
+ * whole.
+ *
+ *  a) This file is free software; you can redistribute it and/or
+ *     modify it under the terms of the GNU General Public License as
+ *     published by the Free Software Foundation; either version 2 of the
+ *     License, or (at your option) any later version.
+ *
+ *     This file is distributed in the hope that it will be useful
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ * Or, alternatively
+ *
+ *  b) Permission is hereby granted, free of charge, to any person
+ *     obtaining a copy of this software and associated documentation
+ *     files (the "Software"), to deal in the Software without
+ *     restriction, including without limitation the rights to use
+ *     copy, modify, merge, publish, distribute, sublicense, and/or
+ *     sell copies of the Software, and to permit persons to whom the
+ *     Software is furnished to do so, subject to the following
+ *     conditions:
+ *
+ *     The above copyright notice and this permission notice shall be
+ *     included in all copies or substantial portions of the Software.
+ *
+ *     THE SOFTWARE IS PROVIDED , WITHOUT WARRANTY OF ANY KIND
+ *     EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ *     OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ *     NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ *     HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY
+ *     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ *     OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include "armada-3720-rz-db.dts"
+/* This file enables eth1 to work in SGMII mode with 1g or 2.5g fixed-link,
+ * eth0 still works in RGMII mode.
+ */
+/ {
+	soc {
+		internal-regs {
+			eth1: ethernet@40000 {
+				phy-mode = "sgmii";
+				status = "okay";
+				fixed-link {
+					/* Here "speed" is set to 1000, GBE MAC is running in 1G mode, */
+					/* but the actual MAC speed may be 1 Gbps or 2.5 Gbps, */
+					/* it's up to the corresponding SERDES speed in u-boot fdt comphy node. */
+					/* If SERDES speed is set to 3.125G in u-boot, it implies sgmii 2.5 Gbps; */
+					/* if SERDES speed is set to 1.25G in u-boot, it implies sgmii 1 Gbps. */
+					speed = <1000>;
+					full-duplex;
+				};
+			};
+			usb3@58000 {
+				status = "disabled";
+			};
+		};
+	};
+};
+
diff --git a/arch/arm64/boot/dts/marvell/armada-3720-rz-db-sgmii1.dts b/arch/arm64/boot/dts/marvell/armada-3720-rz-db-sgmii1.dts
new file mode 100644
index 0000000..644af1b
--- /dev/null
+++ b/arch/arm64/boot/dts/marvell/armada-3720-rz-db-sgmii1.dts
@@ -0,0 +1,65 @@
+/*
+ * Device Tree file for Marvell Armada 3720 RZ development board
+ * (DB-88F3720-DDR3-RZ)
+ * Copyright (C) 2016 Marvell
+ *
+ * Ken Ma <make@marvell.com>
+ *
+ * This file is dual-licensed: you can use it either under the terms
+ * of the GPL or the X11 license, at your option. Note that this dual
+ * licensing only applies to this file, and not this project as a
+ * whole.
+ *
+ *  a) This file is free software; you can redistribute it and/or
+ *     modify it under the terms of the GNU General Public License as
+ *     published by the Free Software Foundation; either version 2 of the
+ *     License, or (at your option) any later version.
+ *
+ *     This file is distributed in the hope that it will be useful
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ * Or, alternatively
+ *
+ *  b) Permission is hereby granted, free of charge, to any person
+ *     obtaining a copy of this software and associated documentation
+ *     files (the "Software"), to deal in the Software without
+ *     restriction, including without limitation the rights to use
+ *     copy, modify, merge, publish, distribute, sublicense, and/or
+ *     sell copies of the Software, and to permit persons to whom the
+ *     Software is furnished to do so, subject to the following
+ *     conditions:
+ *
+ *     The above copyright notice and this permission notice shall be
+ *     included in all copies or substantial portions of the Software.
+ *
+ *     THE SOFTWARE IS PROVIDED , WITHOUT WARRANTY OF ANY KIND
+ *     EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ *     OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ *     NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ *     HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY
+ *     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ *     OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include "armada-3720-rz-db.dts"
+/* This file enables eth1 to work on SERDES SGMII1 interface and
+ * use SMI to do PHY polling, meanwhile eth0 still works on RGMII mode.
+ */
+/ {
+	soc {
+		internal-regs {
+			eth1: ethernet@40000 {
+				phy-mode = "sgmii";
+				phy = <&phy1>;
+				status = "okay";
+			};
+			usb3@58000 {
+				status = "disabled";
+			};
+		};
+	};
+};
+
diff --git a/arch/arm64/boot/dts/marvell/armada-3720-rz-db.dts b/arch/arm64/boot/dts/marvell/armada-3720-rz-db.dts
new file mode 100644
index 0000000..f4560a6
--- /dev/null
+++ b/arch/arm64/boot/dts/marvell/armada-3720-rz-db.dts
@@ -0,0 +1,243 @@
+/*
+ * Device Tree file for Marvell Armada 3720 RZ development board
+ * (DB-88F3720-DDR3-RZ)
+ * Copyright (C) 2016 Marvell
+ *
+ * Gregory CLEMENT <gregory.clement@free-electrons.com>
+ *
+ * This file is dual-licensed: you can use it either under the terms
+ * of the GPL or the X11 license, at your option. Note that this dual
+ * licensing only applies to this file, and not this project as a
+ * whole.
+ *
+ *  a) This file is free software; you can redistribute it and/or
+ *     modify it under the terms of the GNU General Public License as
+ *     published by the Free Software Foundation; either version 2 of the
+ *     License, or (at your option) any later version.
+ *
+ *     This file is distributed in the hope that it will be useful
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ * Or, alternatively
+ *
+ *  b) Permission is hereby granted, free of charge, to any person
+ *     obtaining a copy of this software and associated documentation
+ *     files (the "Software"), to deal in the Software without
+ *     restriction, including without limitation the rights to use
+ *     copy, modify, merge, publish, distribute, sublicense, and/or
+ *     sell copies of the Software, and to permit persons to whom the
+ *     Software is furnished to do so, subject to the following
+ *     conditions:
+ *
+ *     The above copyright notice and this permission notice shall be
+ *     included in all copies or substantial portions of the Software.
+ *
+ *     THE SOFTWARE IS PROVIDED , WITHOUT WARRANTY OF ANY KIND
+ *     EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ *     OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ *     NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ *     HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY
+ *     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ *     OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/dts-v1/;
+
+#include "armada-372x-rz.dtsi"
+
+/ {
+	model = "Marvell Armada 3720 RZ Development Board DB-88F3720-DDR3";
+	compatible = "marvell,armada-3720-db", "marvell,armada3720", "marvell,armada3710";
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+	};
+
+	memory {
+		device_type = "memory";
+		reg = <0x00000000 0x00000000 0x00000000 0x20000000>;
+	};
+
+	soc {
+		internal-regs {
+			spi0: spi@10600 {
+				status = "okay";
+
+				spi-flash@0 {
+					#address-cells = <1>;
+					#size-cells = <1>;
+					compatible = "jedec,spi-nor";
+					reg = <0>; /* Chip select 0 */
+					spi-max-frequency = <104000000>;
+
+					pinctrl-names = "default";
+					pinctrl-0 = <&spi0_pins &spi_quad_pins>;
+
+					partition@0 {
+						label = "Bootloader";
+						reg = <0 0x200000>;
+					};
+					partition@200000 {
+						label = "U-boot Env";
+						reg = <0x200000 0x10000>;
+					};
+					partition@210000 {
+						label = "Linux";
+						reg = <0x210000 0xDF0000>;
+					};
+				};
+			};
+
+			i2c0: i2c@11000 {
+				status = "okay";
+				expander0: pca9555@22 {
+					compatible = "nxp,pca9555";
+					reg = <0x22>;
+					gpio-controller;
+					#gpio-cells = <2>;
+				};
+			};
+
+			i2c1: i2c@11080 {
+				status = "okay";
+			};
+
+			/*
+			* Exported on the micro USB connector CON32
+			* through an FTDI
+			*/
+			uart0: serial@12000 {
+				status = "okay";
+			};
+
+			eth0: ethernet@30000 {
+				phy-mode = "rgmii-id";
+				phy = <&phy0>;
+				status = "okay";
+			};
+
+			mdio@32004 {
+				phy0: ethernet-phy@0 {
+					reg = <0>;
+				};
+				phy1: ethernet-phy@1 {
+					reg = <1>;
+				};
+			};
+
+			/* CON31 */
+			usb3@58000 {
+				status = "okay";
+				usb-phy = <&usb3_phy>;
+			};
+
+			usb@5e000 {
+				status = "okay";
+			};
+
+			interrupt-controller@74840 {
+				status = "okay";
+			};
+
+			sdhci1: sdhci@d0000 {
+				wp-inverted;
+				bus-width = <4>;
+				cd-gpios = <&gpio_sb 2 GPIO_ACTIVE_LOW>;
+				vqmmc-supply = <&vcc_sd_reg1>;
+				status = "okay";
+
+				pinctrl-names = "default";
+				pinctrl-0 = <&sdio_pins &cd_pins>;
+			};
+
+			sdhci0: sdhci@d8000 {
+				xenon,emmc;
+				non-removable;
+				bus-width = <8>;
+				tuning-count = <6>;
+				status = "okay";
+
+				pinctrl-names = "default";
+				pinctrl-0 = <&mmc_pins>;
+			};
+
+			/* CON3 */
+			sata@e0000 {
+				target-supply = <&reg_5v_sata>;
+				status = "okay";
+			};
+			vcc_sd_reg1: regulator@0 {
+				compatible = "regulator-gpio";
+				regulator-name = "vcc_sd1";
+				regulator-min-microvolt = <1800000>;
+				regulator-max-microvolt = <3300000>;
+				regulator-boot-on;
+
+				gpios = <&gpio_sb 22 GPIO_ACTIVE_HIGH>;
+				gpios-states = <0>;
+				states = <1800000 0x1
+					  3300000 0x0>;
+
+				enable-active-high;
+			};
+		};
+
+		reg_sata: pwr-sata {
+			compatible = "regulator-fixed";
+			regulator-name = "pwr_en_sata";
+			regulator-min-microvolt = <12000000>;
+			regulator-max-microvolt = <12000000>;
+			enable-active-high;
+			regulator-boot-on;
+			gpio = <&expander0 2 GPIO_ACTIVE_HIGH>;
+		};
+
+		reg_5v_sata: v5-sata {
+			compatible = "regulator-fixed";
+			regulator-name = "v5.0-sata";
+			regulator-min-microvolt = <5000000>;
+			regulator-max-microvolt = <5000000>;
+			vin-supply = <&reg_sata>;
+		};
+
+		reg_12v_sata: v12-sata {
+			compatible = "regulator-fixed";
+			regulator-name = "v12.0-sata";
+			regulator-min-microvolt = <12000000>;
+			regulator-max-microvolt = <12000000>;
+			vin-supply = <&reg_sata>;
+		};
+
+		pcie-controller {
+			status = "okay";
+
+			pcie@1,0 {
+				reset-gpios = <&gpio_sb 3 GPIO_ACTIVE_LOW>;
+
+				pinctrl-names = "default";
+				pinctrl-0 = <&pcie_pins>;
+
+				status = "okay";
+			};
+		};
+	};
+
+	usb3_phy: usb3-phy {
+		compatible = "usb-nop-xceiv";
+		vcc-supply = <&exp_usb3_vbus>;
+	};
+
+	exp_usb3_vbus: usb3-vbus {
+		compatible = "regulator-fixed";
+		regulator-name = "usb3-vbus";
+		regulator-min-microvolt = <5000000>;
+		regulator-max-microvolt = <5000000>;
+		enable-active-high;
+		regulator-always-on;
+		gpio = <&expander0 1 GPIO_ACTIVE_HIGH>;
+	};
+};
+
diff --git a/arch/arm64/boot/dts/marvell/armada-372x-rz.dtsi b/arch/arm64/boot/dts/marvell/armada-372x-rz.dtsi
new file mode 100644
index 0000000..ceaca13
--- /dev/null
+++ b/arch/arm64/boot/dts/marvell/armada-372x-rz.dtsi
@@ -0,0 +1,63 @@
+/*
+ * Device Tree Include file for Marvell Armada 372x family of RZ SoCs
+ * (also named 88F3720-RZ)
+ *
+ * Copyright (C) 2016 Marvell
+ *
+ * Gregory CLEMENT <gregory.clement@free-electrons.com>
+ *
+ * This file is dual-licensed: you can use it either under the terms
+ * of the GPL or the X11 license, at your option. Note that this dual
+ * licensing only applies to this file, and not this project as a
+ * whole.
+ *
+ *  a) This file is free software; you can redistribute it and/or
+ *     modify it under the terms of the GNU General Public License as
+ *     published by the Free Software Foundation; either version 2 of the
+ *     License, or (at your option) any later version.
+ *
+ *     This file is distributed in the hope that it will be useful
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ * Or, alternatively
+ *
+ *  b) Permission is hereby granted, free of charge, to any person
+ *     obtaining a copy of this software and associated documentation
+ *     files (the "Software"), to deal in the Software without
+ *     restriction, including without limitation the rights to use
+ *     copy, modify, merge, publish, distribute, sublicense, and/or
+ *     sell copies of the Software, and to permit persons to whom the
+ *     Software is furnished to do so, subject to the following
+ *     conditions:
+ *
+ *     The above copyright notice and this permission notice shall be
+ *     included in all copies or substantial portions of the Software.
+ *
+ *     THE SOFTWARE IS PROVIDED , WITHOUT WARRANTY OF ANY KIND
+ *     EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ *     OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ *     NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ *     HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY
+ *     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ *     OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include "armada-37xx.dtsi"
+
+/ {
+	model = "Marvell Armada 3720 RZ SoC";
+	compatible = "marvell,armada3720", "marvell,armada3710";
+
+	cpus {
+		cpu@1 {
+			device_type = "cpu";
+			compatible = "arm,cortex-a53","arm,armv8";
+			reg = <0x1>;
+			enable-method = "psci";
+		};
+	};
+
+};
-- 
1.7.9.5

